src/redux/actions/pipeline/runPipeline.js

```diff
@@ -0,0 +1,54 @@
+import {
+  EXPERIMENT_SETTINGS_PIPELINE_STATUS_LOADING,
+  EXPERIMENT_SETTINGS_PIPELINE_STATUS_ERROR,
+} from '../../actionTypes/experimentSettings';
+
+import getApiEndpoint from '../../../utils/apiEndpoint';
+import loadPipelineStatus from '../experimentSettings/loadPipelineStatus';
+
+const runPipeline = (experimentId, callerStepKey) => async (dispatch, getState) => {
+  dispatch({
+    type: EXPERIMENT_SETTINGS_PIPELINE_STATUS_LOADING,
+    payload: {
+      experimentId,
+    },
+  });
+
+  const processingConfig = getState().experimentSettings.processing[callerStepKey];
+
+  try {
+    const response = await fetch(
+      `${getApiEndpoint()}/v1/experiments/${experimentId}/pipelines`,
+      {
+        method: 'POST',
+        headers: { 'Content-Type': 'application/json' },
+        body: JSON.stringify({
+          processingConfig: [
+            {
+              name: callerStepKey,
+              body: processingConfig,
+            },
+          ],
+        }),
+      },
+    );
```

Comment:
Here it might be good to add a comment that explains why that config is passed (iirc it will automatically merge it into the existing processing config schema in dynamo instead of manually updating it).